streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -128,6 +128,11 @@ public StateStore getStore(final String name) {
         return stateMgr.getStore(name);
     }
 
+    @Override
+    public void initializeTaskTime() {
+        //no-op
```

Comment:
nit: space before `no-op`

Reply:
Oh, tried to run a test without this method, but AbstractTaskTest would break unfortunately if this method is not implemented in AbstractTask (apparently, AbstractTask's constructor is called, and compiler complains about it as a result).

Reply:
Oh, actually, yeah, we can remove this method. We could just modify the AbstractTaskTest itself.

Reply:
If you read the test code you see that the test extends `AbstractTask` in a nested class -- hence you need to add a no-op implementation to the nested class in the test. It should still be removed here.